app.rb

```diff
@@ -4,128 +4,40 @@
 require './book'
 require './classroom'
 require './rental'
+require './lib/create_books'
+require './lib/create_persons'
+require './lib/create_rental'
+require './lib/displayer'
 
 class App
```

Comment:
- [ ] Hi @Debas-31, this class still has `multiple` responsibilities as pointed out by the previous reviewer. Remember the Single Responsibility Principle of **SOLID** states that a class should have `one and only one` reason to change that's a class should have `only` a single rule to play. In this class, in some of the methods, you are still carrying out multiple responsibilities like:
- printing data to the console( check lines 16, 20, 29, 34, 39, 48, 53, 54, 56, 58, 64, 68, 73, 74, 76, 78, 82, 86, 87, 89, 93, 97, 102, 109, 115, 119, 122).
- Getting information from the user (lines 18, 40, 55, 57, 75, 77, 88, 90, 100, 107, 110, 120) and
- Saving data.
This is not in line with the [SRP of SOLID](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid). If you are able to achieve the SRP, the possibility of reusing this class will increase :smiley: .

Reply:
We have made changes according to last reviewer and you checked our deleted code